.agent-os copy/specs/2025-09-04-claude-hook-integration/sub-specs/api-spec.md

```
@@ -0,0 +1,464 @@
+
+**Official Claude Code Hook Input Structure:**
+
+```json
+{
+  "session_id": "abc123",
+  "transcript_path": "/Users/.../claude/projects/.../00893aaf-19fa-41d2-8238-13269b9b3ca0.jsonl",
+  "cwd": "/Users/nathanvale/code/bun-changesets-template",
+  "hook_event_name": "PostToolUse",
+  "tool_name": "Write" | "Edit" | "MultiEdit",
+  "tool_input": {
+    "file_path": "/absolute/path/to/file.ts",
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)